docs/design/imageset-configuration.md

```diff
@@ -0,0 +1,70 @@
+# oc-mirror Imageset Configuration
+===
+
+- [# oc-mirror Imageset Configuration](#-oc-mirror-imageset-configuration)
+- [Content Types](#content-types)
+  - [Platforms](#platforms)
+  - [Operators](#operators)
+  - [Additional Images](#additional-images)
+  - [Helm Chart](#helm-chart)
+- [Limitations](#limitations)
+  - [Platforms and Catalogs](#platforms-and-catalogs)
+  - [Helm Charts](#helm-charts)
+
+
+# Content Types
+
+## Platforms
+
+Container management platforms release channels can be specified for mirroring. 
+Currently, OpenShift Container Platform is supported by all `oc-mirror` commands.
+OKD is a supported type in the imageset configuration, but not by the `oc-mirror` list commands.
+
+## Operators
+
+Catalog images can be specified for mirroring. Bundle images and all the dependencies
+will be mirrored and the catalog with be rebuilt by `oc-mirror` with the custom file-based catalog
```

Comment:
Shouldn't this be written like this: "will be mirrored and the catalog will be rebuilt by `oc-mirror` with the custom file-based catalog" - mind the "catalog will be" instead of "catalog with be".

Reply:
@ableischwitz Thanks for catching this. A fix was merged on a separate PR #443.